metadata.yaml

```diff
@@ -11,6 +11,7 @@ subordinate: false
 series:
   - bionic
   - xenial
+  - cosmic
```

Comment:
If we are adding this for the canary testing, maybe we should leave this out and use --force to deploy canary versions, to avoid marking charms as "supported" on cosmic. What do you think?

Reply:
Yup I like that, I'll look at how that might be implemented.